pages/linux/winetricks.md

```diff
@@ -0,0 +1,16 @@
+# winetricks
+
+> Manage Wine virtual Windows environments.
+> More information: <https://wiki.winehq.org/Winetricks>.
+
+- Start a graphical setup at the default wine location:
+
+`winetricks`
+
+- Specify a wine prefix to run winetricks in:
```

Comment:
The term 'prefix' is quite common across our pages, but I think it conveys the point just as well to use 'directory'. 
```suggestion
- Specify a custom wine directory to run winetricks in:
```